command/sub8_missions/sub8_missions/autonomous.py

```diff
@@ -69,8 +70,9 @@ def do_mission(self):
                         yield self.move.yaw_right_deg(179).go(speed=0.4)
                         yield self.move.yaw_right_deg(179).go(speed=0.4)
 
+            fprint("Vampire Slayer")
```

Comment:
`self.send_feedback` has the advantage of printing these user feedback messages in the client terminal that ran the mission or in the mission GUI, which is probably more useful